.travis.yml

```diff
@@ -1,26 +1,34 @@
+cache: pip
+dist: xenial
 language: python
 python:
   - "2.7"
-  - "3.6"
+  - "3.7"
 
 env:
-  - SPHINX_SPEC=Sphinx~=1.4.0
-  - SPHINX_SPEC=Sphinx~=1.5.0
-  - SPHINX_SPEC=Sphinx~=1.6.0
-  - SPHINX_SPEC=git+https://github.com/sphinx-doc/sphinx.git\#egg\=Sphinx-dev
+  - SPHINX_SPEC='==1.4.0'
+  - SPHINX_SPEC='==1.5.0'
+  - SPHINX_SPEC='==1.6.1'
+  - SPHINX_SPEC='==1.7.0'
+  - SPHINX_SPEC='==1.8.0'
+  - SPHINX_SPEC='==2.0.0'
```

Comment:
Should we add `2.1.x` here, too? Also, doing `~=` would also make sure we're using the latest patch release in each series.